cf_xarray/accessor.py

```diff
@@ -924,7 +921,9 @@ def _rewrite_values(
             value = kwargs[key]
             mappers = all_mappers[key]
 
-            if isinstance(value, str):
+            if isinstance(value, (str, DataArray, Dataset)) or not isinstance(
+                value, Iterable
+            ):
```

Comment:
I made it more general, not sure if it's the best thing to do... (e.g., I don't think currently there are wrapped methods where you can pass a `Dataset`).

Reply:
What if we used this instead... (from https://github.com/pydata/xarray/pull/4972)
``` python
def always_sequence(obj):
    if not isinstance(obj, (list, tuple)):
        obj = [obj]

    return obj
```

Reply:
Should we also add set and dict? Otherwise we are going to return `[{}]`.
Should we place it in `utils.py`?

Reply:
Maybe:
```python
def always_sequence(obj):
    if isinstance(obj, set):
        obj = list(obj)
    elif not isinstance(obj, (list, tuple, dict)):
        obj = [obj]

    return obj
```

Reply:
Yeah that looks right

Reply:
We could call it `always_iterable`